components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__phenotype.xml

```diff
@@ -195,7 +195,7 @@
   // ------------------------------------------------------------------------
   // Behavior of the quick search box
   // ------------------------------------------------------------------------
-  var offsetTop = $('patient-record-actions') &amp;&amp; $('patient-record-actions').getHeight() || 0;
+  var offsetTop = 0;
```

Comment:
The vertical offset causes the "Quick phenotype search box" to become fixed only after scrolling past the input field. Then when it scrolls further, it jumps. Changing offsetTop to 0 causes the "Quick Phenotype search box" to become fixed at the top as soon as the view is past the top of the div making it a smoother transition. Additionally, it doesn't seem possible for the search bar and patient actions toolbar to overlap so I felt the vertical offset wasn't necessary.